paper-dropdown-menu-light.js

```diff
@@ -80,28 +109,6 @@ To style it:
 @hero hero.svg
```

Comment:
This line can be removed. There's also one [in `paper-dropdown-menu.js`](https://github.com/PolymerElements/paper-dropdown-menu/pull/298/files#diff-2951e6f740f646de1ce1b7f94aa0823dR90).